1004-button-group-design-guidance.md

```diff
@@ -0,0 +1,58 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-15
+updated: 2024-10-22
+---
+
+# Button Group Component
+
+The [button group](https://clarity.design/documentation/button-group) component is used to create collections of similar type of action buttons.
+
+## Guidance
+
+Button groups follow normal [button](https://guidance.clarity.design/1003) design guidelines. All button options regarding classes and sizes apply to button groups.
+
+### Button Group Types
+
+- **Must** use `solid` button groups to direct the user's attention to the primary actions the application is suggesting the user take.
+- **Must** use `outline` button groups to indicate a set of secondary actions or to reduce the visual noise on a page.
+- **Must** use `flat` button groups for a set of tertiary actions.
+- **Should** use `flat` button groups in cards and above datagrid components.
+- **May** use `solid`, `outline` and `flat` buttons in normal (default) and small (compact) sizes.
```

Comment:
```suggestion
- **May** use all button types in normal (default) and small sizes.
```

Simplify wording and don't introduce an alternative word for the small button size.